tests/processes/ProcessingTest.py

```diff
@@ -218,6 +218,20 @@ def test_raw_run(self):
                                              'test message',
                                              self.unreadable_path)])
 
+    def test_run_with_aspect(self):
```

Comment:
Actually this test objective is to cover https://github.com/coala/coala/pull/4397/files#diff-20ec48fe3730a3eba01621d03049f9dbR431 which I don't sure which function I can most independently do unit test for (or whether https://github.com/coala/coala/pull/4397/files#diff-20ec48fe3730a3eba01621d03049f9dbR431 is the best place to inject those aspects).

If anyone could point me better way to test those line, maybe we can avoid PickleError (or maybe this PickleError is also a real error that will shown up in normal coala run with aspects in the future).